### ISSUE TITLE
Export identity button completely freezes the app

### ISSUE THREAD

**Issue by Arraxis** [futo-org/grayjay-android#906]
When on the "Manage Policentric identity" screen I press the "Export" button, then I'm taken to an empty black screen and the app freezes completely.

I also tried to copy the long code under my username from the Manage screen and attempted to import it to my new device. It didn't work regardless if I was using the "polycentric://" prefix or not.

I also found the automatically generated .ezip file on my original device but the "Import Grayjay Auto-Backup" option is grayed out on both of my devices.

I'm at my wit's end here and I'm unable to move over my polycentric identity in any way. 

![Screenshot_20240413_011802_Grayjay](https://github.com/futo-org/grayjay-android/assets/166879394/a23ec62a-768f-4010-8cb7-0973f2893726)
![Screenshot_20240413_011657_Grayjay](https://github.com/futo-org/grayjay-android/assets/166879394/54b6f31e-0b36-4dd9-b466-cc3b135a9a31)
![Screenshot_20240413_012933_Grayjay](https://github.com/futo-org/grayjay-android/assets/166879394/d6fd63c0-3297-476d-8196-634a0252a2ce)

**Comment by thapa-sabin** [comment]
I am also getting the exact same issue on my device

**Comment by LastMagmarian** [comment]
I am also experiencing this issue

**Comment by TheWheee** [comment]
I am experiencing this issue.

**Comment by b-risk** [comment]
Same here, was wanting to switch to GrapheneOS right about now when I discovered this issue, hope it is on the top of their list of issues to work on because if you lose your account right now it is unrecoverable

**Comment by thapa-sabin** [comment]
> Same here, was wanting to switch to GrapheneOS right about now when I discovered this issue, hope it is on the top of their list of issues to work on because if you lose your account right now it is unrecoverable

Does the issue still persist? I was able to get it working later on and I am on Graphene OS.

**Comment by b-risk** [comment]
> > Same here, was wanting to switch to GrapheneOS right about now when I discovered this issue, hope it is on the top of their list of issues to work on because if you lose your account right now it is unrecoverable
> 
> Does the issue still persist? I was able to get it working later on and I am on Graphene OS.

Yeah I just tried it, for all I know this could be an account issue because I created my account within a couple days after the app came out.

**Comment by Arraxis** [comment]
> > > Same here, was wanting to switch to GrapheneOS right about now when I discovered this issue, hope it is on the top of their list of issues to work on because if you lose your account right now it is unrecoverable
> > 
> > 
> > Does the issue still persist? I was able to get it working later on and I am on Graphene OS.
> 
> Yeah I just tried it, for all I know this could be an account issue because I created my account within a couple days after the app came out.

That could be. I also created my account very early. Sadly none of the updates fixed the issue so far. 😔

**Comment by b-risk** [comment]
> > > > Same here, was wanting to switch to GrapheneOS right about now when I discovered this issue, hope it is on the top of their list of issues to work on because if you lose your account right now it is unrecoverable
> > > 
> > > 
> > > Does the issue still persist? I was able to get it working later on and I am on Graphene OS.
> > 
> > 
> > Yeah I just tried it, for all I know this could be an account issue because I created my account within a couple days after the app came out.
> 
> That could be. I also created my account very early. Sadly none of the updates fixed the issue so far. 😔

This seems like a very odd issue to be unnoticed, I'm going to try copying the code manually and putting it on a new device first to see if it works and if it does I'm just gonna switch to Graphene.

**Comment by TheWheee** [comment]
It seems to work on harbour tho😪

**Comment by b-risk** [comment]
> > > > > Same here, was wanting to switch to GrapheneOS right about now when I discovered this issue, hope it is on the top of their list of issues to work on because if you lose your account right now it is unrecoverable
> > > > 
> > > > 
> > > > Does the issue still persist? I was able to get it working later on and I am on Graphene OS.
> > > 
> > > 
> > > Yeah I just tried it, for all I know this could be an account issue because I created my account within a couple days after the app came out.
> > 
> > 
> > That could be. I also created my account very early. Sadly none of the updates fixed the issue so far. 😔
> 
> This seems like a very odd issue to be unnoticed, I'm going to try copying the code manually and putting it on a new device first to see if it works and if it does I'm just gonna switch to Graphene.

Update on that, I tried the same thing to copy over the code to my tablet with the "polycentric://" protocol before it and it said it was invalid so the code given in the indentity screen must be something different.
Any help @Kelvin-FUTO ?
